index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task Manager</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://cdn.jsdelivr.net/npm/vue@2/dist/vue.js"></script>
+    <script defer src="app.js"></script>
+</head>
+<body>
+<div id="app">
+    <div class="main-container">
+        <div class="search-container">
+            <form @submit.prevent="submitSearchForm" id="search-task-form">
+                <div class="search-content">
+                    <h3>Search tasks by name:</h3>
+                    <input type="text" name="search-tasks-input" id="search-tasks-input"
+                           required
+                           v-model="searchTasksForm.searchString">
+                    <button>Search</button>
+                </div>
+
+            </form>
+        </div>
+        <div class="new-task-container">
+            <form @submit.prevent="submitNewTaskForm" id="new-task-form">
+                <div class="new-task-content">
+                    <h3>Enter a new task:</h3>
+                    <input type="text" name="new-task-name-input" id="new-task-name-input"
+                           placeholder="Task name"
+                           required
+                           v-model="newTaskForm.taskName">
+                    <input type="text" name="new-task-description-input" id="new-task-description-input"
+                           placeholder="Task description"
+                           required
+                           v-model="newTaskForm.taskDescription">
+                    <button>Submit</button>
+                </div>
+            </form>
+        </div>
+        <div class="tasks-container">
+            <div class="tasks-content">
+                <table class="tasks-table">
+                    <tr>
+                        <th>Task Name</th>
+                        <th>Task Description</th>
+                        <th>Complete Task</th>
+                    </tr>
+
+                    <tr v-for="task in tasks">
```

Comment:
when using v-for we should add a bound key attribute:
see more: https://vuejs.org/v2/guide/list.html#Maintaining-State